File: BibliotecaAthenaeumEmpleado/src/main/java/sv/arrupe/biblioteca/model/Prestamo.java
package sv.arrupe.biblioteca.model;

public class Prestamo {
    public int id_prestamo;
    public int id_lector;
    public String lector_nombre;
    public int id_libro;
    public String libro_titulo;

    public Integer id_empleado_aprueba;
    public String estado;

    public String fecha_solicitud;
    public String fecha_aprobacion;
    public String fecha_vencimiento;
    public String fecha_devolucion;
}

File: Biblioteca_Athenaeum/Models/Prestamo.cs
using System.ComponentModel.DataAnnotations;

namespace Biblioteca_Athenaeum.Models
{
    public class Prestamo
    {
        [Key]
        public int id_prestamo { get; set; }
        public int id_lector { get; set; }
        public int? id_empleado_aprueba { get; set; }
        public int id_libro { get; set; }
        public DateTime fecha_solicitud { get; set; }
        public DateTime? fecha_aprobacion { get; set; }
        public DateTime? fecha_vencimiento { get; set; }
        public DateTime? fecha_devolucion { get; set; }
        public string estado { get; set; }
    }
}